shared/openId4VCI/Utils.ts

```diff
@@ -92,3 +93,19 @@ export const getJWT = async context => {
     throw e;
   }
 };
+export const VC_DOWNLOAD_TIMEOUT =
+  Number(getConfig('openId4VCIDownloadVCTimeout')) || 30000;
```

Comment:
dont need default value 30000 here, as cached response itself will set it, if not avl.

Reply:
If the promise fails to resolve or the key is missing from the server, the promise.catch would return `''` and `Number('')` would evaluate to `0`, hence this or condition is required.

Reply:
Refactored to use getAllConfigurations